output_3.4.txt

```diff
@@ -0,0 +1,8 @@
+
+TASK: 3.4
```

Comment:
+ in this task there should be only one query
+ add query results
+ all there queries are not correct (and, I guess, have different results)